ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -1972,8 +1972,8 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-ipi-disconnected-private
-- as: gcp-ipi-disconnected-private-workload-identity-p2-f14
```

Comment:
I am not sure if it a right to remove `p`, though later we would remove it in batch, cc @liangxia for comments.

Reply:
I'm fine with the changes.